core/components/organisms/select/__stories__/styleType/selectOulined.story.jsx

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import { Select, Row, Column, Label, Card } from '@/index';
+import { action } from '@/utils/action';
+
+// CSF format story - Outlined variant
+export const outlined = () => {
+  const medicineList = [
+    { label: 'Aspirin', value: 'Aspirin' },
+    { label: 'Paracetamol', value: 'Paracetamol' },
+    { label: 'Lisinopril', value: 'Lisinopril' },
+    { label: 'Simvastatin', value: 'Simvastatin' },
+    { label: 'Amoxicillin', value: 'Amoxicillin' },
+    { label: 'Ciprofloxacin', value: 'Ciprofloxacin' },
+    { label: 'Metformin', value: 'Metformin' },
+    { label: 'Omeprazole', value: 'Omeprazole' },
+    { label: 'Diazepam', value: 'Diazepam' },
+    { label: 'Levothyroxine', value: 'Levothyroxine' },
+  ];
+
+  const onSelectHandler = (selectedOption) => {
+    action('selectedOption', selectedOption);
+  };
+
```

Comment:
Please check if we need this example where both the `Select` components appears to be same.

<img width="718" height="383" alt="Screenshot 2025-10-31 at 12 31 23 AM" src="https://github.com/user-attachments/assets/eea2920f-4fcb-4435-8c75-86df5bff0dd2" />

Reply:
This is done purposefully because we wanted to show the select in different background. This has also been updated to Figma file.